massa-node/base_config/config.toml

```diff
@@ -1,96 +1,175 @@
 
 
 [logging]
+    # increase for more info about the node. May reduce your node ability to keep up with the network
     level = 2
 
 [api]
+    # max number of future periods considered during requests
     draw_lookahead_period_count = 10
+    # where to send admin level requests (node management)
     bind_private = "127.0.0.1:33034"
+    # where to send public requests (get data, send operation)
     bind_public = "0.0.0.0:33035"
+    # max number items accepted
     max_arguments = 128
 
 [execution]
+    # path to the initial sequential balance ledger
     initial_sce_ledger_path = "base_config/initial_sce_ledger.json"
 
 [consensus]
+    # max number of previously discarded blocks kept in RAM
     max_discarded_blocks = 100
+    # if a block  is future_block_processing_max_periods periods in the future, it is just discarded
     future_block_processing_max_periods = 100
+    # max number of blocks in the future kept in RAM
     max_future_processing_blocks = 400
+    # max number of blocks waiting for dependencies
     max_dependency_blocks = 2048
+    # number of final periods that must be kept at all time (increse to more resilience to short network disconnections)
```

Comment:
at all times

increase for more resilience

. High values will increase RAM usage.